src/Logout.tsx

```diff
@@ -19,6 +24,26 @@ const Logout: FC = () => {
   useEffect(() => {
     const handleSignOut = async () => {
       if (CLOUD) {
+        if (isFlagEnabled('universalLogin')) {
+          fetch('/api/env/quartz-login-url')
+            .then(response => {
+              response
+                .text()
+                .then(response => {
+                  const redirectUrl = `${response}${CLOUD_LOGOUT_PATH}`
```

Comment:
instead of `response`, maybe we should call this something like `quartzLoginUrl` or something

Reply:
Okay yeah I agree! I'll change that here, and in the LoginPage file 👍